FirstTask.py

```diff
@@ -0,0 +1,147 @@
+<<<<<<< HEAD
+# check balance -> done
+# draw money -> done
+# deposit bills -> done
+# transfeer money -> done
+# type of account -> done
+# compare accout -> done
+=======
+>>>>>>> origin/main
+from random import randint as rand
+from os import system as sys
+from time import sleep
+class Bank:
+    def __init__(self,name,passcode,accNumber,balance,accoutType):
+        self.balance=balance
+        self.accountType=accoutType
+        self.accNumber=accNumber
+        self.name=name
+        self.passcode=passcode
+        self.accNumber=accNumber
+
+    def FullName(self):
+        return self.name
+
+    def accountNumber(self):
+        return self.accNumber
+
+    def Passcode(self):
+        return self.passcode
+
+    def checkBalance(self):
+        return self.balance
+
+    def depositBill(self,bill):
+        self.balance+=bill
+        return True
+
+    def drawMoney(self,money):
+        if(self.balance>=money):
+            self.balance-=money
+            return True
+        else:
+            return False
+    
+    def transferMoney(self,amount):
+        if(self.balance>=amount):
+            self.balance-=amount
+            return True
+        else:
+            return False
+  
+    def checkAccountType(self):
+        return self.accountType
+
+    def accDetails(self):
+        print("Account # ", self.accNumber)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+
+obj = []
+if __name__ == '__main__':
+   while(1):
+        option=int(input('1) Create Account \n2) Login \n0) Exit\nChoose:'))
+        if(option == 1):
+            sys("clear")
+            accno=rand(1,100)+1000
+            name=input('Enter Full Name:')
+            passcode=int(input("Enter Passcode:"))
+            accType=input("Enter Account Type:")

```

Comment:
`accType` should be chosen from a list of given types. Otherwise, the user can enter whatever he likes. 
List of supported account types could be:
- current
- saving
- business